app/src/main/AndroidManifest.xml

```diff
@@ -61,6 +61,7 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".player.exploration.ExplorationActivity"
+      android:configChanges="fontScale"
```

Comment:
We aren't overriding ``onConfigurationChanged``--do we actually need this?

Reply:
Done.